README.md

```diff
@@ -37,5 +31,21 @@ On hovering over a cell:
 - Move the hovered cell up by `20px` (use `transform`)
 - All changes should be animated with the duration of 0.5s
 
+## Checklist
+
+❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
+
+- [DEMO LINK](https://<your_account>.github.io/layout_calendar/)
+- [TEST REPORT LINK](https://<your_account>.github.io/layout_calendar/report/html_report/)
+
+❗️ Copy this `Checklist` to the `Pull Request` description after links, and put `- [x]` before each point after you checked it.
+
+- [ ] Changing 'month-lengh' and 'start-day' modifier in the code element
+reflects in changing calendar layout
+- [ ] Each day has no modifiers, only class (eg. calendar__day)
+- [ ] All `Typical Mistakes` from `BEM` lesson theory are checked.
+- [ ] Code follows all the [Code Style Rules ❗️](./checklist.md)
```

Comment:
We can give a link to the global guideline here
A they already know SCSS

And the same in the checklist
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md